src/compressed_tensors/quantization/utils/helpers.py

```diff
@@ -123,11 +116,6 @@ def calculate_qparams(
 
         zero_points = torch.zeros(scales.shape, device=device, dtype=min_vals.dtype)
     else:
-        if is_fp4(quantization_args=quantization_args):
-            raise NotImplementedError(
-                "Asymmetric Quantization is not supported for FP4"
-            )
-
```

Comment:
Seems like asymmetric FP4 will silently fail without this error? Perhaps if you move the global scale logic to line 123, you can support it?